doc/developers/contributing.rst

```diff
@@ -695,12 +695,14 @@ In general have the following in mind:
     4. 1D or 2D data can be a subset of
        ``{array-like, ndarray, sparse matrix, dataframe}``. Note that ``array-like``
        can also be a ``list``, while ``ndarray`` is explicitly only a ``numpy.ndarray``.
-    5. When specifying the data type of a list, use ``of`` as a delimiter:
+    5. Specific ``dataframe`` when "frame-like" features are being used, such
```

Comment:
I might remove `dataframe` from l. 677 example because in most cases we will not have to specify it then.

Reply:
specific -> specify